packages/create-redwood-app/template/package.json

```diff
@@ -17,5 +17,8 @@
   "engines": {
     "node": "14.x",
     "yarn": "1.x"
+  },
+  "prisma": {
+    "seed": "yarn rw exec seed"
```

Comment:
So the advantage of doing it this way is that we can pass 1) path and 2) handle ts/js, correct?

Reply:
> So the advantage of doing it this way is that we can pass 1) path and 2) handle ts/js, correct?

How seeding now works is that:

* `prisma db seed`
* `prisma migrate reset`
* `prisma migrate dev`

Can also seed because it simple runs any command set in `prisma > seed` in package.json.

Theoretically, this can be any sql script -- well really any script --  too, so @peterp there is a very real, potential Snaplet integration here. 

> When you want to use prisma migrate dev or prisma migrate reset without seeding, you can pass the --skip-seed flag.

See: https://www.prisma.io/docs/guides/database/seed-database#integrated-seeding-with-prisma-migrate

May want to consider adding `--skip-seed` to `migrate dev` but not sure yet.

Reply:
> So the advantage of doing it this way is that we can pass 1) path and 2) handle ts/js, correct?

@peterp I could see a RW `setup snaplet` cli command that "does snaplet stuff" and adds a bash script update and point to the db load.

Reply:
#want

Reply:
> #want

@thedavidprice Do you think it is worth making a `setup seed` command to make the modifications to scripts and package json.

`yarn rw setup seed prisma` would be the default standard

and then could add on more providers later, but the cli would exist.

Reply:
Thought, now all you have to do instead is:

* yarn rw g script snaplet-load
* and then change `"seed": "yarn rw exec seed"` to `"seed": "yarn rw exec snappy-load"`

Reply:
How about this — combining both with an option:
```
yarn rw g script snaplet-load --seed
```

?